syntax = "proto3";
package archive;





/*
DeleteArchive请求
*/
message DeleteArchiveRequest {
    /*
    是否彻底删除文件（true|false）
    */
    string deleteFile = 1 ;
    /*
    是否最后一个版本
    */
    string lastVersion = 2 ;
    /*
    程序包Id
    */
    string packageId = 3 ;
    /*
    版本Id
    */
    string versionId = 4 ;
}

/*
DeleteArchive返回
*/
message DeleteArchiveResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    错误信息
    */
    string error = 2 ;
    /*
    返回信息
    */
    string message = 3 ;
    /*
    返回数据,成功删除的版本Id
    */
    string data = 4 ;
}

/*
DeleteArchiveApi返回
*/
message DeleteArchiveResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    archive.DeleteArchiveResponse data = 4 ;
}

